### DISCUSSION TITLE
Codeium bug with nvim-tree

### DISCUSSION THREAD

**Discussion by jkopano** [LazyVim/LazyVim#5090] [Q&A]
https://github.com/user-attachments/assets/3453b86f-e28a-4663-8f0c-5bff0340909d

From what i understand, codeium does not work well with cmd completions, and also completion are disabled for normal cmdline, but nvim-tree is probably using its own bar for implementing rename/create file etc.

How can i specify that blink completion dont work on certain buffer?

**Comment by johnallen3d** [comment]
FWIW - I've run into this with [avante](https://github.com/yetone/avante.nvim) as well

**Comment by givensuman** [comment]
Also get this issue with oil.nvim
Not sure if there's any reason for the plugin to run when `buftype` is set (e.g. there's no autocompletion in a terminal or prompt window anyways).

**Comment by babalolajnr** [comment]
Same issue I am having but with minifiles
